frontend/src/styles.css

```
@@ -387,3 +387,24 @@
+.request-ai-tabs button{display:flex;align-items:center;justify-content:center;gap:8px;border:0;border-bottom:2px solid transparent;background:transparent;padding:10px 12px;color:var(--muted);font-weight:650}
+.request-ai-tabs button[aria-selected="true"]{border-bottom-color:#29449f;color:#29449f}
+.request-ai-tabs button:focus-visible{outline:2px solid var(--blue);outline-offset:-2px}
+.request-ai-tab-status{display:inline-flex;align-items:center;min-height:18px;border-radius:9px;background:#edf0f4;padding:1px 7px;color:#657286;font-size:9px;font-weight:600;line-height:12px;text-transform:uppercase;letter-spacing:.03em}
+.request-ai-tab-status[data-status="success"]{background:#e1eefb;color:#2c6ca7}.request-ai-tab-status[data-status="error"]{background:#fdeceb;color:#b42318}
+.request-ai-result.request-ai-draft{padding-top:0;border-top:0}
+.request-ai-action{width:24px;height:24px;flex:none;overflow:hidden;margin:0;border:0;border-radius:50%;padding:0}
+.request-document-group .request-ai-action{position:relative;width:24px;height:24px;flex:none;overflow:hidden;margin:0;border:1px solid var(--line);border-radius:50%;background:#fff;padding:0;color:var(--blue)}
+.request-document-group .request-ai-action:hover{background:#f7f8fa;color:#1d3285}
+.request-document-group .request-ai-action:active{transform:scale(.94)}
+.request-document-group .request-ai-action:disabled,.request-document-group .request-ai-action.is-loading{cursor:wait;opacity:.58;transform:none;box-shadow:none}
+.request-ai-pulse{width:16px;height:16px;display:grid;place-items:center;overflow:hidden}.request-ai-pulse svg{width:16px;height:16px;fill:currentColor;stroke:currentColor;stroke-width:1.35;stroke-linecap:round;stroke-linejoin:round}.request-ai-pulse circle{stroke:none}
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)